lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -72,13 +73,14 @@ def self.run_cmd_with_bolt(cmd, nodes)
     # @param task [string] Task to run on nodes
     # @param params [hash] task parameters to send to bolt
     # @param nodes [string] Host or space delimited hosts to run task on
+    # @param modulepath [string] The modulepath to use when running bolt
     #
     # @return [true,false] Based on exit status of the bolt task
-    def self.run_task_with_bolt(task, params, nodes)
+    def self.run_task_with_bolt(task, params, nodes, modulepath = RAS_MODULE_PATH)
```

Comment:
I didn't see a unit test to validate that modulepath got used for this method

Reply:
Added tests to run_task_with_bolt and run_plan_with_bolt